src/reducers/link.reducer.js

```diff
@@ -14,82 +14,130 @@ import { LinkRecord } from '../constants/flowdesigner.model';
 
 const defaultState = new Map();
 
+/**
+ * Delete link information in (In and Out) cache
+ * @params state
+ * @params action
+ */
+function tryDeleteLinkCache(state, action) {
+	let newState = state;
```

Comment:
mmmh right it was true it was an attempt when i started working on this feature but now those check can go away i guess